README.md

```diff
@@ -19,3 +23,31 @@ Folder structure:
   - python
 - ...
 ```
+
+## Building Bahir
+
+Bahir is built using [Apache Maven](http://maven.apache.org/).
+To build Bahir and its example programs, run:
+
+    mvn -DskipTests clean install
+
+## Running Tests
+
+Testing first requires [building Bahir](#building-bahir). Once Bahir is built, tests
```

Comment:
Well, at least currently, it **is** possible to build **and** test the project in **one** shot `mvn clean install`, but this may change in the future and running the build and tests separately is fine as long as developers actually do run the tests ;-)